composer.json

```diff
@@ -133,13 +133,58 @@
         "smarty/smarty": "^4.3.1",
         "soundasleep/html2text": "^0.5.0",
         "swiftmailer/swiftmailer": "~6.2",
+        "symfony/asset": "5.4.*",
+        "symfony/cache": "5.4.*",
+        "symfony/config": "5.4.*",
+        "symfony/console": "5.4.*",
+        "symfony/dependency-injection": "5.4.*",
         "symfony/doctrine-messenger": "^5.4",
+        "symfony/dom-crawler": "5.4.*",
+        "symfony/dotenv": "5.4.*",
+        "symfony/error-handler": "5.4.*",
+        "symfony/event-dispatcher": "5.4.*",
+        "symfony/expression-language": "5.4.*",
+        "symfony/filesystem": "5.4.*",
+        "symfony/finder": "5.4.*",
+        "symfony/form": "5.4.*",
+        "symfony/http-client": "5.4.*",
+        "symfony/http-foundation": "5.4.*",
+        "symfony/http-kernel": "5.4.*",
+        "symfony/inflector": "5.4.*",
+        "symfony/intl": "5.4.*",
+        "symfony/ldap": "5.4.*",
```

Comment:
not sure we need this ldap component

Reply:
Hi @FabienPapet, have a look into the description :) . I'm not removing libraries which are not used but just removing a deprecation. I want to leave the same state than before, with symfony/symfony. We should open a second PR and investigate on which library is not used (and I think there is more than just symfony/ldap 👍  )